### DISCUSSION TITLE
Jump (scroll) to the right / left

### DISCUSSION THREAD

**Discussion by fml2** [sumatrapdfreader/sumatrapdf#2667] [Ideas]
With the keys `PageDown` / `PageUp` it's possible to scroll the document down/up in big steps.

When viewing a document at a big zoom factor, I often find myself missing the possibility to scroll the document to the right/left in big steps. Only smooth scrolling with `Right`/`Left` is possible.

Hence I propose to add a keyboard shortcut for scrolling the document horizontally in bigger steps. I'd find the shortcut `Shift` + `PageUp`/`PageDown` reasonable.

**Comment by GitHubRulesOK** [comment]
it should still be shift Left and Right arrows or H & L but L is Right 🙂  and now you could reset their commands to l <&> r

mouse its only `Shift + Scroll Wheel - Pan horizontally with scroll wheel` so I use dynamic auto pan (Middle button)

**Reply by fml2** [reply]
> it should still be shift Left and Right arrows

Oh, cool, thank you! It's not on the [docs page](https://www.sumatrapdfreader.org/docs/Keyboard-shortcuts).

**Comment by kjk** [comment]
I've updated the docs to mention scrolling with `left / right arrow`, `shift + left / right arrow` and `h` / `l`
